modules/internal/ChapelStandard.chpl

```diff
@@ -26,46 +26,46 @@ module ChapelStandard {
   // Internal, but uses standard/CommDiagnostics
 
   // Internal modules.
```

Comment:
Let's put a `note: these public uses do not make the module name itself available` here